### PR TITLE
PR - Added Opening and Closing section for ComboBox and ChoiceBox

### LINKED ISSUE #67: Document ChoiceBox and ComboBox open/close methods

### PR DESCRIPTION
**Pull request by bbrennanbasis**
This PR closes #67 .
The Opening and Closing sections, added under Max Row Count, also briefly mention that event listeners are available for these actions.

### CODE REVIEW

File: docs/components/lists/combo-box.md [hunk 1]
```diff
@@ -85,6 +85,12 @@ javaC='https://raw.githubusercontent.com/webforj/ControlSamples/main/src/main/co
 height='250px'
 />
 
+## Opening and Closing
```
Comment (hyyan) [inline docs/components/lists/combo-box.md:88]: 'Opening and Closing' should use sentence-style capitalization

File: docs/components/lists/combo-box.md [hunk 2]
```diff
@@ -85,6 +85,12 @@ javaC='https://raw.githubusercontent.com/webforj/ControlSamples/main/src/main/co
 height='250px'
 />
 
+## Opening and Closing
+
+The visibility of the options for a `ComboBox` can be programmatically controlled with the `open()` and `close()` methods.
+These methods allow you to display the list of options for selection or hide it as needed, providing greater flexibility in managing the behavior of a `ComboBox`.
+Additionally, webforJ has event listeners for when the `ComboBox` is closed and when it's opened, giving you more control to trigger specific actions.
```
Comment (hyyan) [inline docs/components/lists/combo-box.md:92]: Add some code snippets

**Review by hyyan — CHANGES_REQUESTED** [review]

File: docs/components/lists/combo-box.md (hunk 1 above)
Comment (bbrennanbasis) [reply docs/components/lists/combo-box.md]: Corrected the capitalization for this section and the rest of the sections.

File: docs/components/lists/combo-box.md (hunk 2 above)
Comment (bbrennanbasis) [reply docs/components/lists/combo-box.md:93]: Here is the following code snippet I've added:

```
//Change the focus to the next component in a form
ComboBox university = new ComboBox("University");
ComboBox major = new ComboBox("Major");
Button submit = new Button("Submit");

//... Add lists of universities and majors

university.onClose( e ->{
  major.focus();
});

major.onClose( e ->{
  submit.focus();
});
```

**Review by hyyan — APPROVED** [review]

**Pull request merged**